src/ToolBox/superpmi/superpmi-shared/methodcontext.cpp

```diff
@@ -3349,7 +3199,7 @@ void MethodContext::dmpGetFieldInfo(const Agnostic_GetFieldInfo& key, const Agno
                 break;
         }
     }
-    printf("}");
+    printf("}\n");
 }
```

Comment:
You are right. Fixed, also deleted trailing newlines from dmpGetInlinedCallFrameVptr.